### PR TITLE
Cleanup code

### PR DESCRIPTION
**Pull request by CjMoore**
- cleans up code in controllers and models

### CODE REVIEW

**Review by meyerhoferc — APPROVED** [review]
looks great, ready for second review & merge @valerietrudell @cmacaulay

File: app/models/item.rb [hunk 1]
```diff
@@ -8,7 +8,7 @@ class Item < ApplicationRecord
   has_many :orders, through: :order_items
   has_many :reviews
 
-  has_attached_file :image, styles: {medium: "200x200", thumb: "100x100"}, default_url: "/images/dumbledore.jpg" # "http://vignette3.wikia.nocookie.net/harrypotter/images/4/40/Albus_Dumbledore_%28HBP_promo%29_3.jpg/revision/latest/scale-to-width-down/700?cb=20150822232849"
+  has_attached_file :image, styles: {medium: "200x200", thumb: "100x100"}, default_url: "/images/dumbledore.jpg"
```
Comment (cmacaulay) [inline app/models/item.rb:11]: Great catch!

File: app/views/shared/_items.html.erb [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
+            <% else %>
+              <button  class="btn disabled">Retired</button>
+            <% end %>
+        </div>
       </div>
     </div>
-  </div>
-<% end %>
+  <% end %>
 </div>
```
Comment (cmacaulay) [inline app/views/shared/_items.html.erb:26]: 🎉

**Review by cmacaulay — COMMENTED** [review]
Looks great!

**Pull request merged**